### PR TITLE
fix(config): Add repo-rules and update .gitignore

### LINKED ISSUE #3: Track repo-rules
At the moment this repo is more general purpose. I'd like to make this repo specific to fCC. The first step is to include the rules in git so that PR's can be made towards them.

```json
{
  "freeCodeCamp/freeCodeCamp": {
    "repoContribPath": "/blob/staging/CONTRIBUTING.md",
    "userBlacklistForPR": [
        "greenkeeperio-bot",
        "QuincyLarson",
        "BerkeleyTrue"
    ],
    "actions": [
      "opened",
      "reopened",
      "synchronize"
    ],
    "rules": {
      "critical": {
        "blacklistedBaseBranchNames": [
          "master"
        ],
        "blacklistedHeadBranchNames": [
          "master",
          "staging"
        ]
      },
      "allowedBranchNames": [
        "fix/",
        "feature/",
        "translate/"
      ],
      "closeKeywords": [
        "close",
        "closes",
        "closed",
        "fix",
        "fixes",
        "fixed",
        "resolve",
        "resolves",
        "resolved"
      ],
      "maxCommitCount": 1
    }
  }
}
```

### PR DESCRIPTION
**Pull request by raisedadead**
This commit adds the repo-rules.json for freeCodeCamp and updates the .gitignore to official node
ignores

Closes #3

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 require('dotenv')
   .load({path: path.resolve(path.dirname(__filename), '../.env')});
-require('./middlewars').checkCredentials();
```
Comment (Greenheart) [inline lib/utils.js:4]: wars? :scream:

Comment (raisedadead) [reply lib/utils.js:4]: Yeah, I have corrected the typo. Right?

**Comment by BerkeleyTrue** [comment]
cc @berkeleytrue

File: lib/utils.js (hunk 1 above)
Comment (Greenheart) [reply lib/utils.js:4]: Hehe yeah, good job! :wink:

**Comment by BerkeleyTrue** [comment]
@raisedadead Thanks!

**Pull request merged**